CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
-- None
+### 2.2.0 - 2019-04-05
+#### Changed
+- Added the ability to remove Authorization headers from any HTTP request made when fetching files not hosted on Stripe.
+- Refactored LocalFile.js to improve readability.
```

Comment:
Please remove the references to the next version and the date, as well as the changed header.
Unreleased should just read like a list of changes. When I'm prepping to publish on npm, I'll handle that kind of formatting in the CHANGELOG.

When you merge in the latest from master, you'll be able to resolve this when you resolve the merge conflict for this file since Brock added some new things recently.